ProcessLib/LIE/HydroMechanics/CreateHydroMechanicsProcess.cpp

```diff
@@ -282,6 +283,24 @@ std::unique_ptr<Process> createHydroMechanicsProcess(
                 "time-dependent.",
                 frac_prop->aperture0.name.c_str());
         }
+
+        auto permeability_model_config =
```

Comment:
This part can be put to a creator function, e.g
createFracturePermeability(...)
Then the path to the doc can be changed as that in my previous comment.